hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestSnapshotDeletingService.java

```diff
@@ -188,6 +191,30 @@ public void testMultipleSnapshotKeyReclaim() throws Exception {
     assertTableRowCount(deletedTable, 0);
 
     verifySnapshotChain(delSnapInfo, null);
+
+    // verify the cache of purged snapshot
+    // /vol1/bucket2/bucket2snap1 has been cleaned up from cache map
+    SnapshotCache snapshotCache = om.getOmSnapshotManager().getSnapshotCache();
+    Method getDbMapMethod = snapshotCache.getClass()
+        .getDeclaredMethod("getDbMap");
+    getDbMapMethod.setAccessible(true);
+    ConcurrentHashMap<String,
+        ReferenceCounted<IOmMetadataReader, SnapshotCache>> dbMap =
+        (ConcurrentHashMap<String,
+            ReferenceCounted<IOmMetadataReader,
+                SnapshotCache>>) getDbMapMethod.invoke(snapshotCache);
+
+    assertEquals(2, dbMap.size());
+
+    Method getPendingEvictionListMethod =
```

Comment:
Similar to previous comment, may be create a public function for `pendingEvictionList#size`.

Reply:
yeah, that makes sense! updated! Thanks @hemantk-12 !